### PR TITLE
feat/array-methods-demo

### PR DESCRIPTION
**Pull request by PReichetanz**
Add demo for JS Array Methods session.

rendered: https://github.com/neuefische/web-exercises/tree/array-methods-demo/sessions/js-array-methods

## CSB
- [Link to Demo Start](https://codesandbox.io/s/github/neuefische/web-exercises/tree/array-methods-demo/sessions/js-array-methods/demo-start?file=/js/index.js)

- [Link to Demo End](https://codesandbox.io/s/github/neuefische/web-exercises/tree/array-methods-demo/sessions/js-array-methods/demo-end?file=/js/index.js)

refers to neuefische/web-curriculum-new-format#22

### CODE REVIEW

File: sessions/js-array-methods/demo-end/css/styles.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+h1,
+h2 {
+	color: orangered;
+}
+
+article {
+	margin: 2rem;
+	padding: 0.5rem;
+	border: 1px solid black;
+}
```
Comment (CJAkkuec) [inline sessions/js-array-methods/demo-end/css/styles.css:22]:
```suggestion
:root {
  --nemo: #ff4a11;
  --granite: #252629;
  --water: #f3f5f9;
  --foam: #ffffff;
}

*,
*::before,
*::after {
  box-sizing: border-box;
}

body {
  margin: 0;
  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica,
    Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
}

h1,
h2 {
  color: var(--nemo);
}

article {
  margin: 2rem;
  padding: 0.5rem;
  border-radius: 10px;
  background: var(--water);
}
```
An alternative styling _suggestion_ featuring neuefische colors/color names. :)

File: sessions/js-array-methods/demo-start/css/styles.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+h1,
+h2 {
+	color: orangered;
+}
+
+article {
+	margin: 2rem;
+	padding: 0.5rem;
+	border: 1px solid black;
+}
```
Comment (CJAkkuec) [inline sessions/js-array-methods/demo-start/css/styles.css:22]:
```suggestion
:root {
  --nemo: #ff4a11;
  --granite: #252629;
  --water: #f3f5f9;
  --foam: #ffffff;
}

*,
*::before,
*::after {
  box-sizing: border-box;
}

body {
  margin: 0;
  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica,
    Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
}

h1,
h2 {
  color: var(--nemo);
}

article {
  margin: 2rem;
  padding: 0.5rem;
  border-radius: 10px;
  background: var(--water);
}
```
An alternative styling _suggestion_ featuring neuefische colors/color names. :)

**Review by CJAkkuec — COMMENTED** [review]
Love this idea ;) just a few notes

**Review by Kheirah — APPROVED** [review]
Great demo! Should be fun to teach

**Pull request merged**